bciers/apps/dashboard/middlewares/withAuthorizationDashboard.tsx

```diff
@@ -24,22 +24,22 @@ const isUnauthenticatedAllowListedPath = (pathname: string): boolean => {
 };
 
 // Function to check if the path is in the authenticated allow list
-const isAuthenticatedAllowListedPath = (pathname: string) => {
-  const allowList = [dashboard, "registration", "reporting"];
-
-  // Split the pathname into segments
-  const segments = pathname.split("/");
-
-  // Iterate through each segment
-  for (const segment of segments) {
-    // Check if any value in the allowList matches the current segment
-    if (allowList.includes(segment)) {
-      return true; // Return true if match found
-    }
-  }
-
-  return false; // Return false if no match found
-};
+// const isAuthenticatedAllowListedPath = (pathname: string) => {
```

Comment:
Was this a temporary comment out that needs to go back in?

Reply:
Well this was an unused variable so it caused a linting failure once we got that working again. I don't know if it's needed in the future so I just commented it out unlike most of the unused variables.

Reply:
Gotcha. I think this was probably replaced by `isUnauthenticatedAllowListedPath` when @shon-button and @joshgamache were working on auth